src/aiovodafone/api.py

```diff
@@ -414,23 +414,39 @@ async def _reset(self) -> bool:
 
         return False
 
-    async def _login_json(self, username: str, password: str) -> bool:
+    async def _login_json(
+        self, username: str, password: str, useChallenge: bool = False
+    ) -> bool:
         """Login via json page"""
 
-        payload = {
-            "LoginName": username,
-            "LoginPWD": password,
-        }
+        if useChallenge:
+            return_dict = await self._get_sercomm_page("/data/login.json")
```

Comment:
Please create a method that return the correct payload to make it easier to read